lib/livebook/evaluator/io_proxy.ex

```diff
@@ -289,6 +311,28 @@ defmodule Livebook.Evaluator.IOProxy do
     end
   end
 
+  defp chars_from_input("", _count), do: {:eof, ""}
+
+  defp chars_from_input(input, count) do
+    case byte_size(input) do
+      size when size >= count ->
+        chars = binary_part(input, 0, count)
```

Comment:
Note that this gets `count` number of bytes, but unicode characters may take multiple utf8 bytes. See [this StringIO bit](https://github.com/elixir-lang/elixir/blob/8bec593f5dab77b2f6eb0a197fdabcf81a7ffd05/lib/elixir/lib/string_io.ex#L308-L338) for the reference :) To test that just include some emojis like 🚀 or other fancy characters in the input.

Reply:
What I meant was that `count` refers to the number of characters and not bytes, so this needs to be updated similarly to the StringIO version.

Reply:
Got it, I'll check the possibilities, maybe use something like `String.length`

Reply:
We need to count codepoints and length works with graphemes. The linked code is all you need:

https://github.com/elixir-lang/elixir/blob/d9028a8d0405d19076418376ab50bdc83ff72974/lib/elixir/lib/string_io.ex#L308-L338

:)

Reply:
I used the links as a reference and now `getn` accepts special characters 😃